src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1522,7 +1523,8 @@ internal static bool ReportInvalidScopedOverrideAsError(MethodSymbol baseMethod,
             {
                 var baseParameter = baseParameters[i];
                 var overrideParameter = overrideParameters[i + overrideParameterOffset];
-                if ((baseParameter.RefKind, overrideParameter.RefKind) is (RefKind.RefReadOnlyParameter, RefKind.In) or (RefKind.In, RefKind.RefReadOnlyParameter))
+                if ((baseParameter.RefKind, overrideParameter.RefKind) is (RefKind.RefReadOnlyParameter, RefKind.In) or (RefKind.In, RefKind.RefReadOnlyParameter) ||
+                    (methodGroupConversion && OverloadResolution.IsRefMismatchAcceptableForMethodConversion(baseParameter.RefKind, overrideParameter.RefKind, compilationOpt: null)))
```

Comment:
Consider clearly separating what mismatch rules are applied when:
```methodGroupConversion ?
        OverloadResolution.IsRefMismatchAcceptableForMethodConversion(baseParameter.RefKind, overrideParameter.RefKind, compilationOpt: null) :
        ((baseParameter.RefKind, overrideParameter.RefKind) is (RefKind.RefReadOnlyParameter, RefKind.In) or (RefKind.In, RefKind.RefReadOnlyParameter))```

Reply:
BTW, do we care what kind of mismatch is this? Can we simplify the check to just ```baseParameter.RefKind != overrideParameter.RefKind```?